packages/block-library/src/navigation/style.scss

```diff
@@ -230,7 +230,8 @@
 // Menu items with no background.
 .wp-block-navigation .wp-block-page-list,
 .wp-block-navigation__container {
-	gap: 0.5em 2em;
+	row-gap: calc(var(--wp--style--block-gap, 2em) / 4);
```

Comment:
A thought, since we are thinking about a block spacing control that in the future allows you to swap between uniform and axial gap, perhaps we should keep the single `gap` property, but just switch between the singular and shorthand syntax?

From [CSS Tricks](https://css-tricks.com/almanac/properties/g/gap/):

```
.container {
  gap: 1rem;
  /* Is equivalent to:
  *  row-gap: 1rem;
  *  column-gap: 1rem
  */
  
  gap: 10px 15%;
  /* Is equivalent to:
  *  row-gap: 10px;
  *  column-gap: 15%;
  */
}
```

<img width="1012" alt="Spacing" src="https://user-images.githubusercontent.com/1204802/135582805-000315b8-80b8-4b81-b4ff-36deea7a0578.png">

Reply:
> A thought, since we are thinking about a block spacing control that in the future allows you to swap between uniform and axial gap, perhaps we should keep the single gap property, but just switch between the singular and shorthand syntax?

Thanks for bringing the uniform/axial controls to our attention.

I went with longhand just to make things explicit for me at the time, while I was thinking of how to implement decent defaults. 

Though shorthand is just as good 👍 

I was approaching the task with a view of trying to maintain the current defaults:

```
// Defaults to gap: 0.5em 2em where --wp--style--block-gap isn't available.
.wp-block-navigation__container {
    gap: calc(var(--wp--style--block-gap, 2em) / 4) var(--wp--style--block-gap, 2em);
}
```

When axial eventually becomes a requirement, I see how this won't work however. 

Did you have a method of switching between the singular and shorthand syntax in mind?

My first thought is that we could move the logic to the backend (either by introducing new CSS vars or inline styles). It would make things easier to calculate conditionally.

Reply:
> Did you have a method of switching between the singular and shorthand syntax in mind?

Actually, sorry, just saw the ongoing discussions here:

https://github.com/WordPress/gutenberg/issues/34529

cc @stacimc @andrewserong who have been thinking deeply about this topic.

Reply:
Yep the idea is that you'll be able to switch between uniform or axial gap, with most (all?) blocks defaulting to uniform gap. 

Embracing the singular `gap` value might also be good for global styles overrides, since you could use a single value to support either. But the inner workings here are a bit unclear for me still, so I'm happy to defer.